library/recon/client/payload.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Awesome recon payload. Unfortunately we'll need it in its own directory before it can be accepted. If you can put it in library/recon/client/[your_payload_name]/payload.sh -- that'll do the trick.